src/bosh_aws_cpi/lib/cloud/aws/instance_manager.rb

```diff
@@ -63,14 +66,15 @@ def find(instance_id)
 
     private
 
-    def babysit_instance_creation(instance, vm_cloud_props)
+    def babysit_instance_creation(instance, vm_cloud_props, private_ip_addresses)
       begin
         # We need to wait here for the instance to be running, as if we are going to
         # attach to a load balancer, the instance must be running.
         instance.wait_until_exists
         instance.wait_until_running
         instance.update_routing_tables(vm_cloud_props.advertised_routes)
         instance.disable_dest_check unless vm_cloud_props.source_dest_check
+        attach_ip_prefixes(instance, private_ip_addresses)
```

Comment:
for consistency with the other methods, should this also be a method on `instance`?

Reply:
yes, makes sense